dockers/ubuntu-14.04/Dockerfile

```diff
@@ -84,11 +84,10 @@ RUN curl -sLk https://sourceforge.net/projects/swig/files/swig/swig-4.0.2/swig-4
  && rm -rf swig-4.0.2
 
 # Install Miniconda
```

Comment:
I can't make a suggestion for all these lines, so describing my suggestion just in words.

Let's be explicit here and don't call Mambaforge installation as "Miniconda"/"conda".
Please update lines 86-95 making it clear we are working with `mamba[forge]`. The only public entrypoint here is `CONDA` env variable, so don't change its name.

Reply:
sure, works for me!

updated in https://github.com/guolinke/lightgbm-ci-docker/pull/22/commits/f75b7a4d4c0874bbf5255b5fa75218219bf29870 and tested that building the image still works